src/features/common/Layout/MyForestContextV2.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  FC,
+  createContext,
+  useContext,
+  useMemo,
+  useState,
+  useEffect,
+} from 'react';
+import { trpc } from '../../../utils/trpc';
+import { SetState } from '../types/common';
+import { ErrorHandlingContext } from './ErrorHandlingContext';
+import { useUserProps } from './UserPropsContext';
+import {
+  handleError,
+  APIError,
+  TreeProjectClassification,
+  CountryCode,
+  ProjectPurpose,
+  UnitTypes,
+} from '@planet-sdk/common';
+import { Point } from 'geojson';
+import {
+  ContributionStats,
+  MyContributionsMapItem,
+  MapLocation,
+} from '../types/myForestv2';
+
+interface ProjectInfo {
+  allowDonations: boolean;
+  classification: TreeProjectClassification;
+  country: CountryCode;
+  geometry: Point;
+  guid: string;
+  image: string;
+  name: string;
+  purpose: ProjectPurpose;
+  slug: string;
+  tpoName: string;
+  unitType: UnitTypes;
+}
+interface RegistrationGeojson {
+  geometry: Point | undefined;
+  properties: MyContributionsMapItem | undefined;
+}
+
+interface DonationGeojson {
+  geometry: Point;
+  properties: {
+    projectInfo: ProjectInfo;
+    contributionInfo: MyContributionsMapItem | undefined;
```

Comment:
Here `contributionInfo` is incorrectly typed, as it can only contain `MyContributionsSingleProject`

Reply:
Resolved in 92ebd1a